index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+  <head bgcolor="#B36666">
+      <title>My Test Page </title>
+  </head>
+  <body bgcolor="#FFCCCC">
+	 <section id="Header">
+	    <Header>
```

Comment:
HTML tags are *usually* totally lowercase. It's not a *big* deal, but it messed up the syntax highlighting in my editor, so I figured I'd pass it on. :computer: